api/src/test/resources/fakeextensions.json

```
@@ -960,6 +960,8 @@
       "guide" : "https://quarkus.io/guides/rest-json",
       "categories" : [ "web" ],
       "status" : "stable",
+      "supported-by" : "Red Hat",
```

```suggestion
      "supported-by" : "redhat",
```

or red-hat, not sure which one is recommended for a key syntax

---

It looks like YAML can handle spaces in the key values (feels weird), but this would generate a tag of `redhat:stable` so you wouldn't have the nice looking "Red Hat" in the web page.  Does a test need to be added to prove the spaces will work, and if so, where would that need to be added?

---

Made suggested changes.